TLM/TLM/Manager/Impl/LaneConnection/ConnectionDataBase.cs

```diff
@@ -128,17 +128,17 @@ internal class ConnectionDataBase : Dictionary<LaneEnd, LaneConnectionData[]> {
 
             return ret;
         }
-        
+
         /// <summary>
         /// removes all connections from and to the given lane.
         /// </summary>
         internal void RemoveConnections(uint laneId) {
             var laneStart = new LaneEnd(laneId, true);
             var laneEnd = new LaneEnd(laneId, false);
-            if(this.TryGetValue(laneStart,out var startConnections)) { 
+            if(this.TryGetValue(laneStart,out var startConnections)) {
                 foreach (var connection in startConnections) {
                     uint laneId2 = connection.LaneId;
-                    ushort nodeId = laneId.ToLane().GetNodeId(true /* StartConnections */); 
+                    ushort nodeId = laneId.ToLane().GetNodeId(true /* StartConnections */);
```

Comment:
what is this /* StartConnections */ is this a parameter name? C# syntax has parameter names supported

Reply:
notice I only removed a space at the end. It's a question to @kianzarrin